web3_proxy/src/frontend/errors.rs

```diff
@@ -459,6 +460,17 @@ impl Web3ProxyError {
                     },
                 )
             }
+            Self::InvalidUserTier => {
+                warn!("InvalidUserTier");
+                (
+                    StatusCode::INTERNAL_SERVER_ERROR,
+                    JsonRpcErrorData {
+                        message: Cow::Borrowed("UserTier is not valid!"),
+                        code: StatusCode::BAD_REQUEST.as_u16().into(),
```

Comment:
Is this an internal server error, or a bad request?

Reply:
it is currently called when there is no downgrade_tier_id (while it is required), so it should be an internal server error

Reply:
ok. the code should change from BAD_REQUEST to INTERNAL_SERVER_ERROR

But downgrade_tier_id is not required, so I think we can just delete this error